css/home.css

```diff
@@ -69,29 +70,31 @@ body {
 }
 
 .homepage-hero__article {
-  margin-right: 57px;
-  margin-left: 57px;
+  margin: auto;
+}
+
+.homepage-hero__hdlp-logo{
   position: absolute;
   bottom: 250px;
-  top: 250px;
+  margin-left: -125px;
+  width: 247px;
```

Comment:
Remember to add the size of the logo for desktop, which is `350px`. Add it inside the already existing media querie for desktop.